electrumx/server/env.py

```diff
@@ -27,15 +27,18 @@ class Env(EnvBase):
     # Peer discovery
     PD_OFF, PD_SELF, PD_ON = range(3)
 
-    def __init__(self):
+    def __init__(self, coin=None):
```

Comment:
Please add a docstring explaining why coin can be supplied (external software wanting to provide a coin not in the distribution)